Add examples to plugin docs (#97)
jenkinsci/pipeline-input-step-plugin: README.md

# Pipeline: Input Step Plugin

Adds the Pipeline `input` step to wait for human input or approval. 
A basic Proceed or Abort option is provided in the stage view. 

The parameter entry screen can be accessed via a link at the bottom of the build console log or via link in the sidebar for a build.

Input Example with `Message Parameter`:

  input 'Proceed or Abort'

Input Example with `Custom ID`:
Every input step has an unique ID. It is used in the generated URL to proceed or abort.

A specific ID could be used, for example, to mechanically respond to the input from some external process/tool.

   input id: '2', message: 'input-message'

Input Example with `OK Button Caption`(Optional):

   input message: 'input-message', ok: 'OK'

Input Example with `Allowed Submitter`:

> User IDs and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed. This means that "alice, bob, blah " is the same as "alice,bob,blah".

Reviewer: ```suggestion
User IDs and/or external group names of a person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
```
Changed the sentence structure to be a bit more clear on what will happen, added in a few words to make things read smoothly
